circuits/cpp/src/aztec3/circuits/kernel/private/native_private_kernel_circuit_init.test.cpp

```diff
@@ -297,7 +297,8 @@ TEST_F(native_private_kernel_init_tests, private_function_incorrect_storage_cont
     // Assertion checks
     EXPECT_TRUE(builder.failed());
     EXPECT_EQ(builder.get_first_failure().code, CircuitErrorCode::PRIVATE_KERNEL__CONTRACT_ADDRESS_MISMATCH);
-    EXPECT_EQ(builder.get_first_failure().message, "Storage contract address must be that of the called contract");
+    EXPECT_EQ(builder.get_first_failure().message,
+              "contract address in the call_context must be that of the called contract");
```

Comment:
maybe still important to mention "storage" since storage address may differ from address where code lives?